src/Microsoft.AspNet.Server.Kestrel/Http/Frame.Templated.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Do we have a standard way of of naming generic classes that have a non-generic counterpart?